Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-contests.tsx

```diff
@@ -36,13 +36,15 @@ interface IContestsContext {
         sortingTypes: ISort[];
         pagesInfo: IPagesInfo;
         currentPage: number;
+        contest: IIndexContestsType;
     };
     actions: {
         reload: () => Promise<void>;
         clearFilters: () => void;
         clearSorts: () => void;
         toggleParam: (param: IFilter | ISort) => void;
         changePage: (pageNumber: number) => void;
+        getContestByProblemId: (problemId: number) => void;
```

Comment:
`loadContestByProblemId`. `get...` implies a result is returned, but this is not the case.

Reply:
Renamed.